www/js/lib/util.js

```diff
@@ -318,14 +241,11 @@ define([], function() {
         removeDuplicateStringsInSmallArray: removeDuplicateStringsInSmallArray,
         endsWith: endsWith,
         readIntegerFrom4Bytes: readIntegerFrom4Bytes,
-        readIntegerFrom2Bytes : readIntegerFrom2Bytes,
-        readFloatFrom4Bytes : readFloatFrom4Bytes,
-        uint8ArrayToHex : uint8ArrayToHex,
-        uint8ArrayToBase64 : uint8ArrayToBase64,
-        readFileSlice : readFileSlice,
+        readIntegerFrom2Bytes: readIntegerFrom2Bytes,
+        readFloatFrom4Bytes: readFloatFrom4Bytes,
+        uint8ArrayToHex: uint8ArrayToHex,
+        readFileSlice: readFileSlice,
         binarySearch: binarySearch,
-        b64toBlob: b64toBlob,
-        uintToString: uintToString,
```

Comment:
I removed those unused functions that have to do with converting between uint(8)Arrays and strings, as they are obsolete.